5_People/career.md

```diff
@@ -22,41 +24,35 @@ This is the Salary Formula:
 
 > Note: all amounts are gross and per-month.
 
-### Base Salary
-
-Niteo Base Salary is calculated by first taking the [*Glassdoor salary for the role in San Francisco*](https://www.glassdoor.com/Salaries/san-francisco-software-engineer-salary-SRCH_IL.0,13_IM759_KO14,31.htm), converting it to EUR and finally discounting it by our *affordability ratio*. 
-
-#### Why San Francisco?
+#### Base Salary
 
-We take San Francisco because we assume it has the best data for the tech sector.
+Niteo Base Salary is calculated by first taking the [*Salary.com salary base for the role in United States*](https://www.salary.com/research), converting it to EUR and finally discounting it by our *affordability ratio*. 
 
-#### But what about EUR/USD fluctuations?
+##### What about EUR/USD fluctuations?
 
-To avoid day-to-day currency fluctuations we take [the 10-year exchange rate average](https://www.ofx.com/en-au/forex-news/historical-exchange-rates/yearly-average-rates/) to convert the Glassdoor amounts from USD to EUR, Niteo's primary currency. The exchange rate for 2019 is _1.23_.
+To avoid day-to-day currency fluctuations we take [the 10-year exchange rate average](https://www.ofx.com/en-au/forex-news/historical-exchange-rates/yearly-average-rates/) to convert the Salary.com amounts from USD to EUR, Niteo's primary currency. The exchange rate for 2020 is _1.20_.
 
-#### Affordability Ratio
+##### Affordability Ratio
 
-We are not a VC funded Sillicon Valey startup who scales for the sake of scaling and often does not care how much things cost. We strive to be profitable so [every Nitean gets a cut](https://github.com/niteoweb/handbook/blob/master/5_People/profit-sharing.md). Hence, we discount Glassdor amounts by an *affordability ratio*, which we get by averaging Numbeo Location Ratios of Amsterdam, Barcelona, Berlin, and Lisbon. The affordability ratio for 2019 is _0.53_.
+We are not a VC funded Sillicon Valey startup who scales for the sake of scaling and often does not care how much things cost. We strive to be profitable so [every Nitean gets a cut](https://github.com/niteoweb/handbook/blob/master/5_People/profit-sharing.md). Hence, we discount Salary.com amounts by an *affordability ratio*. The affordability ratio for 2020 is _0.49_.
 
-#### How often do base salaries get calculated?
+##### How often do base salaries get calculated?
 
 Once a year, in January, during our [bi-annual IRL](https://blog.niteo.co/irls/). Same for the affordability ratio.
 
+#### Location Factor
 
-### Location Factor
+Location factor takes into account the living costs. We determine them by comparing Numbeo's [cost of living between United States and the Nitean's country](https://www.numbeo.com/cost-of-living/compare_cities.jsp?country1=United+States&city1=San+Francisco%2C+CA&country2=Germany&city2=Berlin) and calculating the *Numbeo Location Ratio*. 
```

Comment:
Location Factors are compressed towards our affordability ratio: https://github.com/niteoweb/salarycalc/pull/44/files#diff-0f1be8eec416e71c28b41efb0544df26R121

Reply:
The link is comparing cities, it should compare countries.